modules/katello_product.py

```diff
@@ -113,7 +128,7 @@
     'name': 'name',
     'organization': 'organization',
     'description': 'description',
-    # 'gpg_key': 'gpg_key',  # wait for Nailgun
+    'content_credential': 'gpg_key',
```

Comment:
I have not followed the Nailgun change closely, but is this translation still valid/needed?

Reply:
On a second thought, I think in the Product/Repository case, you can specify a _gpg key_, a _client cert_, a _server cert_ and a _ca cert_.

So i think we should still call it a `gpg_key` here and add the ssl parts with their names later.
Sorry for bringing that up so late.